env.example

```diff
@@ -1,8 +1,22 @@
-export LOCATION=eastus
+export LOCATION=westeurope
 export ARO_IMAGE=arointsvc.azurecr.io/aro:latest
 export NO_CACHE=false
 export AZURE_EXTENSION_DEV_SOURCES="$(pwd)/python"
 
-export ARO_SKIP_PKI_TESTS=true
 
-. secrets/env
+# Change this to your preferred OpenShift version
+export OPENSHIFT_VERSION="4.12.25" 
```

Comment:
We could set the version here to 4.13.40 to match the default version in pkg/util/version/const.go, since that is what we're referencing for people to get the hash to use for the PULLSPEC value.

Or a link to quay.io/openshift-release-dev/ocp-release and a note to filter the tags to find the release you're looking for.